pandas/io/excel.py

```diff
@@ -400,9 +400,15 @@ def __init__(self, io, **kwds):
                     pass
 
             data = io.read()
-            self.book = xlrd.open_workbook(file_contents=data)
```

Comment:
you can just wrap this into a kwargs, adding only if its not None, then pass ``**kwargs`` on construction.

Reply:
I agree. I did a new commit on the matter.